config/config_util.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"encoding/json"
+	"reflect"
+)
+
+// CloneConf deeply clones this config.
+func CloneConf(conf *Config) (*Config, error) {
+	content, err := json.Marshal(conf)
+	if err != nil {
+		return nil, err
+	}
+	var clonedConf Config
+	if err := json.Unmarshal(content, &clonedConf); err != nil {
+		return nil, err
+	}
+	return &clonedConf, nil
+}
+
+var (
+	// dynamicConfigItems contains all config items that can be changed during runtime.
+	dynamicConfigItems = []string{
+		"Performance.MaxProcs", "Performance.MaxMemory",
+		"Performance.CrossJoin", "Performance.FeedbackProbability",
+		"Performance.QueryFeedbackLimit", "Performance.PseudoEstimateRatio",
+		"OOMAction", "MemQuotaQuery", "TiKVClient.StoreLimit",
+	}
+)
+
+// MergeConfigItems is used to overwrite some config items that can't be changed during runtime.
```

Comment:
Is this comment wrong? It seems that `overwrite` means `cover`.

MergeConfigItems overwrites the dynamic config items and leaves the other items unchanged?